mp/game/momentum/momentum.fgd

```diff
@@ -14,10 +14,12 @@
 
 @SolidClass base(Targetname, EnableDisable, Toggle) = func_nogrenades : "Rockets and grenades will not detonate inside this area." 
 [
-    airborne_only(choices) : "Prevent airborne explosions only" : 0 = 
+    sticky_fizzle_type(choices) : "Stickybomb prevention method" : 0 =
     [
-        0 : "False"
-        1 : "True"
+        0 : "Fizzle on detonation."
+        1 : "Fizzle on detonation only if the sticky is airborne."
+        2 : "Fizzle if the stickybomb lands inside the trigger."

```

Comment:
Automatically destroy stickybomb upon landing (instead of attempted detonation).